.github/workflows/delete-old-branches.yaml

```diff
@@ -0,0 +1,18 @@
+name: Cleanup old branches
+on:
+  push:
+    branches:
+      - master
```

Comment:
does this need to be dynamic? we use main a lot these days right?

Reply:
oh this is in the actions repo. makes sense.

Reply:
Yeah exactly this file is it running in this repository, just one instance of the action. I wanted to add it here for testing and as an example